tests/pipeline-all-amps/repo/pom.xml

```diff
@@ -0,0 +1,285 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>pipeline-repo</artifactId>
+  <packaging>pom</packaging>
+
+    <parent>
+        <groupId>org.alfresco</groupId>
+        <artifactId>pipeline-all-amps</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+    </parent>
+
+  <properties>
+    <image.name>alfresco/alfresco-pipeline-all-amps-repo</image.name>
+
+    <alfresco.alfresco-rm-enterprise-rest-api-explorer.version>3.0.2</alfresco.alfresco-rm-enterprise-rest-api-explorer.version>
+  </properties>
+
+<dependencyManagement>
+  <dependencies>
+    <dependency>
```

Comment:
If any of these exist in the top level pom, remove them from here.

Reply:
Done